sdk/deploy.go

```diff
@@ -126,6 +147,26 @@ func (d DeployClient) Trigger(projectId string, cfg DeployConfig) (DeployRespons
 	return response, nil
 }
 
-func getDeployEndpoint(projectId string) string {
-	return fmt.Sprintf("api/deploy/projects/%s/trigger/pipeline/", projectId)
+// GetDeployStatus interacts with Mia Platform APIs to retrieve selected pipeline status
+func (d DeployClient) GetDeployStatus(projectId string, pipelineId int, environment string) (StatusResponse, error) {
+	var statusEndpoint string
+	if environment != "" {
+		statusEndpoint = fmt.Sprintf("/api/deploy/projects/%s/pipelines/%d/status/?environment=%s", projectId, pipelineId, environment)
```

Comment:
The status path could be extracted in a constant instead of duplicate it

Reply:
Or, maybe better, it should be possible to set the initial value of status endpoint directly when defined:

```go
statusEndpoint := fmt.Sprintf("/api/deploy/projects/%s/pipelines/%d/status/", projectId, pipelineId)
```

and, if environment is not empty, add the query param.
In this way, the else statement could be removed

Reply:
Done